report/scripts/lighthouse-report.js

```diff
@@ -69,10 +75,22 @@ LighthouseReport.prototype = {
     window.print();
   },
 
+  updateView: function() {
+    if (this.toggleViewUser.checked) {
+      this.viewUserFeature.classList.remove('hidden');
```

Comment:
[Nope, everywhere](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden).

AFAIK it's implemented as the user-agent stylesheet `*[hidden] { display: none; }`.